tests/main/nvidia-files/task.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+set -xeu
+case "$1" in
+prepare)
+	# Install Nvidia userspace libraries at the designated version.
+	apt-get install -y \
+		libnvidia-common-"$PACKAGE_VERSION" \
+		libnvidia-compute-"$PACKAGE_VERSION":amd64 \
+		libnvidia-compute-"$PACKAGE_VERSION":i386 \
+		libnvidia-decode-"$PACKAGE_VERSION":amd64 \
+		libnvidia-decode-"$PACKAGE_VERSION":i386 \
+		libnvidia-encode-"$PACKAGE_VERSION":amd64 \
+		libnvidia-encode-"$PACKAGE_VERSION":i386 \
+		libnvidia-fbc1-"$PACKAGE_VERSION":amd64 \
+		libnvidia-fbc1-"$PACKAGE_VERSION":i386 \
+		libnvidia-gl-"$PACKAGE_VERSION":amd64 \
+		libnvidia-gl-"$PACKAGE_VERSION":i386
+
+	tests.cleanup defer apt-get remove --purge -y \
```

Comment:
This is not needed, this automatically done in prepare-restore.sh. All the packages installed during the test are automatically removed, even the dependencies which are not explicitly installed

Reply:
Yes, I know, this is a leftover from the moment where the multi-arch support code was broken. I'll get rid of it.